crates/swc_ecma_minifier/tests/compress/fixture/projects/wmr/archive-1/chunks/meta-tags.c6b3dbd5/output.js

```diff
@@ -1,7 +1,7 @@
 import { m } from "../index.f66dda46.js";
 import { u as useLang, a as useTitleTemplate, b as useTitle, c as useMeta, d as useLink } from "./hoofd.module.6c5395cb.js";
 function MetaTags() {
-    return useLang("nl"), useTitleTemplate("%s | \u{1F4AD}"), useTitle("Welcome to hoofd"), useMeta({
+    return useLang("nl"), useTitleTemplate("%s | 💭"), useTitle("Welcome to hoofd"), useMeta({
```

Comment:
I think this can be problematic

Reply:
hm, it was original code, so we keep them as is, original code:
```
useTitleTemplate("%s | 💭")
```

Because target is not es5, it will not be problem